ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixParticipant.java

```diff
@@ -147,6 +183,36 @@ public void close() {
     }
   }
 
+  /**
+   * Get the list of sealed replicas from the HelixAdmin. This method is called only after the helixAdministrationLock
```

Comment:
The statement "this method is only called after the helixAdministrationLock is taken" is not true now that this is a public method. Perhaps that statement is unnecessary.

Reply:
@pnarayanan @jsjtzyy There was never a "get" before the current change right? (I don't think the get needs a lock, does it?) The put still needs to be under a lock - maybe this is just a copy-paste error?

Reply:
@vgkholla yeah, my bad. will fix it.